### PR TITLE
Added ability to recognise/use split labs to linfo

### PR DESCRIPTION
**Pull request by Alpheeexx**
pls test dont trust me

### CODE REVIEW

File: core/bin/linfo [hunk 1]
```diff
@@ -116,7 +119,13 @@ parseCmdLine() {
...
    done
-   
+
+   CONF_LIST="$1"
+   while [ ${#} -gt 0 ]
+   do
+      CONF_LIST="$CONF_LIST $2"
+      shift
+   done   
    if [ $# -gt 0 ]; then
```
Comment (AdamBromiley) [inline core/bin/linfo:129]: This `if` statement becomes redundant now that the command allows non-option arguments

**Comment by AdamBromiley** [comment]
The specification of additional `lab.conf`s is inconsistent with the `-d` option (specify the lab directory). Rather, it'd make more sense for the user to input additional lab directories, then check if each directory has a `lab.conf` when the mapping function is later called (as per the original script).

Even better, for consistency, shift it so all directories can be specified in (single or multiple) `-d` options. Or remove the `-d` option and make every directory specifiable as a non-option argument. This will require changes to other parts of `linfo` that rely on the `-d` argument, and may introduce inconsistencies with other Netkit scripts that allow a directory to be specified (like in `lstart`).

Finally, it should allow absolute and relative paths (in the commit they *must* be relative to `$LAB_DIRECTORY`). This is solvable with running the `makeAbsolutePath` helper script on each directory (will mean the lines where `$CONF_PATH` is used in `buildAffectedVHostList` will need tweaking too).

File: core/bin/linfo [hunk 2]
```diff
@@ -52,14 +52,22 @@ This command displays information about a Netkit lab without running it.
 Displayed information includes the number of virtual machines that make up the
 lab. If asked to, the command can create a PNG file containing a graphical map
 of the link-level topology of the lab.
-Available options are:
+Multiple lab configurations can be specified by listing lab.conf files (current required)
+using the -d option. A -d is needed per directory. The -a option will
```
Comment (TechSupportJosh) [inline core/bin/linfo:56]: Improve wording here, does -d specify the directory or the lab.conf file?

File: core/bin/lstart [hunk 3]
```diff
@@ -369,7 +369,7 @@ update_available() {
 		latest_version=$(cat ${NETKIT_HOME}/.new-version)
 		current_version=$(sed "s/Netkit version //g" ${NETKIT_HOME}/netkit-version)
 		
-		if [ ${latest_version} != ${current_version} ]; then
+		if [ "${latest_version}" != "${current_version}" ]; then
```
Comment (TechSupportJosh) [inline core/bin/lstart:372]: Please try to keep different changes in different PRs :-)
